src/FRP/Radius.purs

```diff
@@ -0,0 +1,28 @@
+module FRP.Radius where
+import Prelude
+import FRP.Live
+import FRP (FRP)
+import FRP.Behavior (Behavior, animate, fixB, integral', derivative')
+import FRP.Behavior.Mouse (buttons, position)
+import FRP.Behavior.Time (millisSinceEpoch)
+
+dotFollow = live $ map (\{ x, y } -> dot x y 50.0) mouse
```

Comment:
I don't really see the value in duplicating this here. I'd rather remove them from the readme and move the comments here, or just remove this entirely.

Reply:
I'm fine with merging the readme changes though, if you want to separate them out in the meantime.